components/datetime/src/neo_skeleton.rs

```diff
@@ -1021,45 +960,17 @@ impl NeoComponents {
         }
     }
 
-    /// Whether this field set contains the hour.
-    pub fn has_hour(self) -> bool {
-        match self {
-            NeoComponents::Date(_) => false,
-            NeoComponents::CalendarPeriod(_) => false,
-            NeoComponents::Time(time_components) => time_components.has_hour(),
-            NeoComponents::Zone(_) => false,
-            NeoComponents::DateTime(_, time_components) => time_components.has_hour(),
-            NeoComponents::DateZone(_, _) => false,
-            NeoComponents::TimeZone(time_components, _) => time_components.has_hour(),
-            NeoComponents::DateTimeZone(_, time_components, _) => time_components.has_hour(),
-        }
-    }
-
-    /// Whether this field set contains the minute.
-    pub fn has_minute(self) -> bool {
+    /// Whether this field set contains the time of day.
+    pub fn has_time(self) -> bool {
         match self {
```

Comment:
nit: can be written as `matches!`

Reply:
I prefer `match` because it forces exhaustivity